tests/Translators/Shared/test_active_tim.py

```diff
@@ -0,0 +1,59 @@
+import pytest
+from unittest.mock import patch, MagicMock
+import Translators.Shared.active_tim as active_tim
+
+@pytest.fixture
+def sample_geojson():
+    return {
+        "features": [
+            {
+                "properties": {
+                    "id": "incident_1",
+                    "routeName": "route_1",
+                    "type": "accident",
+                    "laneImpacts": ["lane1"],
+                    "additionalImpacts": ["impact1"]
+                },
+                "geometry": {
+                    "type": "Point",
+                    "coordinates": [100.0, 0.0]
+                }
+            },
+            {
+                "properties": {
+                    "id": "incident_2",
+                    "routeName": "route_2",
+                    "type": "construction",
+                    "laneImpacts": ["lane2"],
+                    "additionalImpacts": None
+                },
+                "geometry": {
+                    "type": "LineString",
+                    "coordinates": [[100.0, 0.0], [101.0, 1.0]]
+                }
+            }
+        ]
+    }
+
+
+@patch('Translators.Shared.active_tim.query_db')
+def test_active_tim_active(mock_query_db):
```

Comment:
(non-blocking) Should we have a corresponding test for the case where the result is False?